src/dqe.erl

```diff
@@ -17,12 +17,12 @@
                          Resolution :: pos_integer()}].
 
 -type query_error() :: {'error', 'no_results' |
-                        'significant_figures' |
-                        'resolution_conflict' |
+                        %% 'significant_figures' |
+                        %% 'resolution_conflict' |
```

Comment:
The dialyzer error that I get otherwise is that `query_error() is a supertype of succession type`.  But because `dflow:recv` is defined `{ok, [term()]}`, the dialyzer no longer has the ability to verify the above definition is actually correct.
I will revert this change, as discussed.